### PR TITLE
☂️ added codecov

### PR DESCRIPTION
**Pull request by bmiddha**
- Added codecov to track test coverage

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -10,13 +10,22 @@
         "serve": "node dist/server.js",
         "serve-debug": "nodemon --inspect dist/server.js",
         "start": "npm run serve",
-        "test": "jest --runInBand --coverage --verbose --detectOpenHandles",
+        "test": "jest -i",
```
Comment (anarmawala) [inline api/package.json:13]: Why is this necessary?

Comment (bmiddha) [reply api/package.json:13]: added jest config in `package.json`  for verbose and converage. -i is same as runinband

**Review by anarmawala — APPROVED** [review]

File: api/package.json (hunk 1 above)
Comment (anarmawala) [reply api/package.json:13]: Understood
